### PR TITLE
Integration icons updates

### LINKED ISSUE #1760: [Website] Integrations icon updates
#### Current State of the Meshery.io site

Homepage: https://meshery.io/

Some application icons of integrations are missing and being default icon.

#### Desired State of the Meshery.io site

Review Integrations icons and updates.

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by shubhusion**
**Description**

This PR fixes #1760

**Notes for Reviewers**

- Changed icons for apisix and gitlab.
- Added missing icons for wordpress-operator , aws-ecs and keda-http-scaler.
- Some icons like GitHub, kubesphere, and Prometheus appear twice with the same permalinks in the integration icon grid on the homepage. Should I remove one of them?

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by shubhusion** [comment]
@leecalcote why is your commit occuring in my PR ?

File: Gemfile.lock
Comment (vishalvivekm) [inline Gemfile.lock:1]: Please any changes related to Gemfile.lock and integration/data.js @shubhusion

**Review by vishalvivekm — CHANGES_REQUESTED** [review]

File: Gemfile.lock
Comment (shubhusion) [reply Gemfile.lock:1]: all changes related to Gemfile.lock were reverted and are now same as master branch of meshery.io

File: Gemfile.lock [hunk 1]
```diff
@@ -112,4 +112,4 @@ DEPENDENCIES
   wdm (>= 0.1.0)
 
 BUNDLED WITH
-   2.4.21
+   2.4.21
```
Comment (sudhanshutech) [inline Gemfile.lock:115]: do not commit this❌

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
merge conflicts

**Comment by sudhanshutech** [comment]
@vishalvivekm are doing any manual changes to integrations, no right?

File: Gemfile.lock (hunk 1 above)
Comment (shubhusion) [reply Gemfile.lock:115]: This Commit was done to restore back to the original installation settings. System made changes to the gemfile while doing the setup.

Comment (leecalcote) [reply Gemfile.lock:115]: @shubhusion in other words, there is no _actual_ change here. Ok, that's good. 

Ideally, this change is removed from this PR as it's unnecessary churn.

Comment (shubhusion) [reply Gemfile.lock:115]: Yeah

**Pull request closed without merging**